graphql-cli/src/main/java/io/ballerina/graphql/generator/ballerina/FunctionBodyGenerator.java

```diff
@@ -172,19 +163,124 @@ public FunctionBodyNode generateInitFunctionBody(AuthConfig authConfig) {
      *
      * @param authConfig the object instance representing authentication configuration information
      * @return           the object instance representing http client config
+     * http:ClientConfiguration httpClientConfig = {auth: config.auth, httpVersion: config.httpVersion,
+     * timeout: config.timeout, forwarded: config.forwarded, poolConfig: config.poolConfig,
+     * compression: config.compression, circuitBreaker: config.circuitBreaker, retryConfig: config.retryConfig,
+     * validation: config.validation};
+     * do {
+     *     if config.http1Settings is ClientHttp1Settings {
+     *         ClientHttp1Settings settings = check config.http1Settings.ensureType(ClientHttp1Settings);
+     *         httpClientConfig.http1Settings = {...settings};
+     *     }
+     *     if config.http2Settings is http:ClientHttp2Settings {
+     *         httpClientConfig.http2Settings = check config.http2Settings.ensureType(http:ClientHttp2Settings);
+     *     }
+     *     if config.cache is http:CacheConfig {
+     *         httpClientConfig.cache = check config.cache.ensureType(http:CacheConfig);
+     *     }
+     *     if config.responseLimits is http:ResponseLimitConfigs {
+     *         httpClientConfig.responseLimits = check config.responseLimits.ensureType(http:ResponseLimitConfigs);
+     *     }
+     *     if config.secureSocket is http:ClientSecureSocket {
+     *         httpClientConfig.secureSocket = check config.secureSocket.ensureType(http:ClientSecureSocket);
+     *     }
+     *     if config.proxy is http:ProxyConfig {
+     *         httpClientConfig.proxy = check config.proxy.ensureType(http:ProxyConfig);
+     *     }
+     * } on fail var e {
+     *     return error graphql:HttpError("GraphQL Client Error", e, body = ());
+     * }
```

Comment:
Let's update the comment replacing `graphql` module instead of `http` module

Reply:
Ack